Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.yml

```diff
@@ -0,0 +1,113 @@
+category: Messaging
+commonfields:
+  id: Mail Listener v2
+  version: -1
+configuration:
+- display: Mail Server Hostname / IP address
+  name: MailServerURL
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: '143'
+  display: IMAP Port
+  name: port
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- defaultvalue: INBOX
+  display: Incoming mail folder
+  name: folder
+  required: true
+  type: 0
+- display: Fetch mails from these senders addresses only (eg. admin@demo.com,test@demo.com)
+  name: permittedFromAdd
+  required: false
+  type: 0
+- display: Fetch mails from these domains only (eg. demo.com, test.com)
+  name: permittedFromDomain
+  required: false
+  type: 0
+- defaultvalue: 3 days
+  display: First fetch time
+  name: first_fetch
+  required: false
+  type: 0
+- defaultvalue: '50'
+  display: The maximum number of incidents to fetch each time
+  name: limit
+  required: false
+  type: 0
+- defaultvalue: 'true'
+  display: Filter by SINCE date for IMAP that supports it. Lotus Notes does not by
+    default.
+  name: filterByDate
+  required: false
+  type: 8
+- display: Delete processed emails
+  name: delete_processed
+  required: false
+  type: 8
+- display: Include raw body in incidents
+  name: Include_raw_body
+  required: false
+  type: 8
+- display: Save the email .eml file
+  name: save_file
+  required: false
+  type: 8
+- display: Use TLS for connection (defaults to True)
```

Comment:
why `defaults to True` ?

Reply:
No particular reason, seems like a better default then False, remove it?

Reply:
yes, like `insecure`